### PR TITLE
adrv9001 project fixes and features

### PR DESCRIPTION
**Pull request by buha**
- .json profile (same profiles as on linux)
- fix iio_app initialization, it was broken
- fix axi_dac / axi_adc configuration

Tested in both normal and ADRV9002_RX2TX2 mode with both CMOS and LVDS profiles+HDL.

### CODE REVIEW

File: projects/adrv9001/src/app/headless.c [hunk 1]
```diff
@@ -81,47 +89,44 @@ int get_sampling_frequency(struct axi_adc *dev, uint32_t chan,
...
 	struct iio_data_buffer iio_dac_buffers[IIO_DEV_COUNT];
 	struct iio_data_buffer iio_adc_buffers[IIO_DEV_COUNT];
 	struct iio_device *iio_descs[IIO_DEV_COUNT * 2];
 	struct iio_app_device app_devices[IIO_DEV_COUNT * 2] = {0};
-	int32_t i, ret;
+	int32_t d, ret;
+	int32_t a; // linear iterator for iio_descs and app_devices flat arrays
 
-	for (i = 0; i < IIO_DEV_COUNT; i++) {
+	for (d = 0; d < IIO_DEV_COUNT; d++) {
```
Comment (amiclaus) [inline projects/adrv9001/src/app/headless.c:200]: why replacing `i` with `d`? If it has a specific meaning, I'd add a comment similar to the one for `a`

**Review by amiclaus — APPROVED** [review]
One minor question on my side.

File: projects/adrv9001/src/app/headless.c (hunk 1 above)
Comment (buha) [reply projects/adrv9001/src/app/headless.c:200]: i reverted that rename - it was indeed not needed

**Comment by buha** [comment]
V2:
- forgot to include the json profiles, fixed
- reverted i->d useless iterator rename

**Comment by buha** [comment]
v3:
- fixed bad refactoring

**Review by dbogdan — APPROVED** [review]

**Pull request merged**